sycl/test-e2e/bindless_images/read_1D.cpp

```diff
@@ -1,7 +1,7 @@
-// REQUIRES: cuda
+// REQUIRES: cuda || (level_zero && gpu-intel-dg2)
```

Comment:
Doesn't this require a specific driver version? Should that be documented somewhere?